src/v2/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -50,36 +51,35 @@ export interface PaymentProps {
   isCommittingMutation: boolean
 }
 
-interface PaymentState {
-  isGettingCreditCardId: boolean
-}
+type Props = PaymentProps & StripeProps
 
 const logger = createLogger("Order/Routes/Payment/index.tsx")
 
-@track((props: PaymentProps) => ({
```

Comment:
Not sure if this is the correct way to add @track to a functional component. Any ideas on this?

Reply:
I think for tracking functional components, we can use the [`useTracking`](https://github.com/NYTimes/react-tracking#usage-with-react-hooks) hook. So the decorator here for class components can just be removed.

Reply:
Okay so we can remove this and we don't need to add the `useTracking` hook in it's place? I don't see the `@track` being used in the component anywhere, so I assume that would be the case.